### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by swarajpure**
This CHANGELOG will maintain a list of changes we do in our apps in terms of features, fixes, documentation, styling changes etc.
After every PR gets merged to main, we'll update the CHANGELOG. For the time being, I'll do it. Once people get a hang of it, we'll start involving the PR authors to do this step.

Now since we're just starting this (it's already quite late, but okay), I have put the current working on my-site in v1.0.0, feel free to point out if I've missed something :)

### CODE REVIEW

**Review by rohan09-raj — COMMENTED** [review]
You missed the new navbar feature.

**Comment by swarajpure** [comment]
> You missed the new navbar feature.

It has not gone to production yet, right?

**Comment by rohan09-raj** [comment]
> > You missed the new navbar feature.
> 
> It has not gone to production yet, right? CHANGELOG will only contain stuff which is released on production

Oh ok I missed that `after every PR gets merged to main`

**Comment by swarajpure** [comment]
> After every PR gets merged to main, we'll update the CHANGELOG. For the time being, I'll do it. Once people get a hang of it, we'll start involving the PR authors to do this step.

No worries! 😄

**Review by rohan09-raj — APPROVED** [review]
Looks good to me 🔥

**Comment by akshay1502** [comment]
`loading spinner for routes.` @swarajpure

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# CHANGELOG.md
+
+## v1.0.0
```
Comment (akshay1502) [inline CHANGELOG.md:3]: If we are going to maintain app changelog as 1.0.0 then every new feature should be 1.x (Like navbar) and sub changes into it should be 1.x.x (like 11.1.aligning nav links 1.1.2. position of GitHub icon/rds logo). Just a suggestion.

Comment (swarajpure) [reply CHANGELOG.md:3]: We'll be following [semantic versioning](https://semver.org/):
```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

For example, v1.2.3, here is 1 stands for MAJOR, 2 for MINOR and 3 for PATCH

Comment (swarajpure) [reply CHANGELOG.md:3]: For more info: https://www.baeldung.com/cs/semantic-versioning

**Comment by akshay1502** [comment]
Semver explained using illustrations guide https://medium.com/fiverr-engineering/major-minor-patch-a5298e2e1798

File: CHANGELOG.md (hunk 1 above)
Comment (swarajpure) [reply CHANGELOG.md:3]: With nice examples and easy to understand explanation: https://medium.com/fiverr-engineering/major-minor-patch-a5298e2e1798

**Pull request merged**